token-lending/program/src/lib.rs

```diff
@@ -21,3 +22,28 @@ pub const NULL_PUBKEY: solana_program::pubkey::Pubkey =
         11, 193, 238, 216, 208, 116, 241, 195, 55, 212, 76, 22, 75, 202, 40, 216, 76, 206, 27, 169,
         138, 64, 177, 28, 19, 90, 156, 0, 0, 0, 0, 0,
     ]);
+         
+/// whitelist liquidator wallets
+pub const WHITELIST_LIQUIDATORS : [solana_program::pubkey::Pubkey; 3] = [
+    // F4q2bm6k4AW2rgQMBfvoNnfn1xFkoD98We6uJd6tV7tq
+    solana_program::pubkey::Pubkey::new_from_array([
+        208, 254, 170,  18, 176, 113,  47,  78,
+        0, 176, 134,  93, 161,  19,  98, 161,
+        114, 185, 140, 222,  45, 158, 143, 251,
+        53, 114,  83, 154, 201, 207, 178, 198
+    ]),
+    // owoD8aRRvKZXRfDiGvXYc18gfQ5cQBcKgEXFk6PCTva
+    solana_program::pubkey::Pubkey::new_from_array([
+        12,   6, 173,  17, 230,  35, 81, 149,
+        182, 240, 121,  19, 165, 180, 38, 237,
+        34,  50, 245,   8, 230, 123, 18, 122,
+        114,  60, 166, 205, 191, 247, 88, 195
+    ]),
+    // 8i3ufSbnCDi3ZGyGJxDco26AQTGB5G81G1SBsUD55mK6  this is for tests
+    solana_program::pubkey::Pubkey::new_from_array([
```

Comment:
This account has [on-chain transactions](https://solscan.io/account/8i3ufSbnCDi3ZGyGJxDco26AQTGB5G81G1SBsUD55mK6) from a year ago. Is that account's private key leaked?

Reply:
it's literally in the repo so yeah

Reply:
its the account they use for tests in repo